service/accounting/db/query.sql

```diff
@@ -17,23 +17,27 @@ RETURNING *;
 
 -- name: UpdateCompanyChartOfAccount :one
 UPDATE accounting.company_chart_of_accounts
-SET account_code = $2,
-account_name = $3,
-account_group = $4,
-bank_name = $5,
-bank_account_number = $6,
-bank_code = $7,
-opening_balance = $8,
-is_deleted = $9,
-updated_at = NOW()
+SET 
+    account_code = $2,
+    account_name = $3,
+    account_group = $4,
+    bank_name = $5,
+    bank_account_number = $6,
+    bank_code = $7,
+    opening_balance = $8,
+    is_deleted = $9,
+    updated_at = NOW()
 WHERE id = $1
 RETURNING *;
 
 -- name: GetCompanyChartOfAccounts :many
 SELECT id, company_id, branch_id, account_code, account_name, account_group, bank_name, 
 bank_account_number, bank_code, opening_balance, is_deleted
 FROM accounting.company_chart_of_accounts
-WHERE company_id = $1 AND account_name LIKE $2;
+WHERE company_id = $1 AND account_name LIKE $2
+AND account_group LIKE $3 
+AND CASE WHEN $4 = '0' THEN is_deleted = FALSE 
```

Comment:
ini and nya ngapain ya?

Reply:
Ini ada 3 kemungkinan bisa filter all, yang sudah deleted, yang masih aktif

Reply:
ini tolong diganti kalau false / true langsung is_deleted = $4, kalau ga is_deleted = is_deleted, karena engga semua bahasa pemrograman ngitung kalau 1 == true -> true

Reply:
https://github.com/kyleconroy/sqlc/discussions/364

Reply:
Ok2 noted